scripts/gitlab/check_runtime.sh

```diff
@@ -93,16 +91,14 @@ then
 
 	# okay so now need to fetch the substrate repository and check whether spec_version or impl_version has changed there
 	SUBSTRATE_CLONE_DIR="$(mktemp -t -d substrate-XXXXXX)"
-	trap "rm -rf ${SUBSTRATE_CLONE_DIR}" INT QUIT TERM ABRT EXIT
-
+	trap 'rm -rf "${SUBSTRATE_CLONE_DIR}"' INT QUIT TERM ABRT EXIT
 
 	git clone --branch polkadot-master --depth 100 --no-tags \
```

Comment:
would be more consistent to use the same `${GIT_DEPTH:-100}` as above

Reply:
also the `polkadot-master` branch doesn't seem to be maintained anymore

Reply:
Cheers, done. Thanks for spotting the old ref to polkadot-master :)